pyalgostrategypool/aaroon_crossover_v2.py

```diff
@@ -0,0 +1,152 @@
+            if self.main_order_map.get(instrument) is None:
+
+                # Get crossover value
+                crossover = self.get_crossover_value(instrument)
+
+                # define key values for action
+                action_constants = {1: 'BUY', -1: 'SELL'}
+
+                if crossover in [-1, 1]:
+                    # Add instrument to the bucket
+                    selected_instruments.append(instrument)
+
+                    # Add additional info for the instrument
+                    meta.append({'action': action_constants[crossover]})
+
+        # Return the buckets to the core engine.
+        # Engine will now call strategy_enter_position with each instrument and its additional info one by one
+        return selected_instruments, meta
+
+    def strategy_enter_position(self, candle, instrument, meta):
+        """
+        This method is called once for each instrument from the bucket in this candle.
+        Place an order here and return it to the core.
+        """
+
+        # Place buy order
+        self.main_order_map[instrument] = _ = self.broker.OrderRegular(instrument, meta['action'], quantity=self.number_of_lots * instrument.lot_size)
+        return _
+
+    def strategy_select_instruments_for_exit(self, candle, instruments_bucket):
+        """
+        This method is called once every candle time.
+        If you set the candle at 5 minutes, then this method will be called every 5 minutes (09:15, 09:20, 09:25 and so on).
+        In a candle, the exit method is called first, then the entry method is called.
+        So once a candle starts, this method strategy_select_instruments_for_exit gets called first
+        and then strategy_select_instruments_for_entry gets called.
+        """
+
+        # Add instrument in this bucket if you want to place an (exit) order for it,
+        # We decide whether to place an instrument in this bucket or not based on the decision-making process given below in the loop
+        selected_instruments = []
+
+        # Add accompanying info in this bucket for that particular instrument in the bucket above
+        meta = []
+
+        for instrument in instruments_bucket:
+            main_order = self.main_order_map.get(instrument)
+
+            # Compute various things and get the decision to place an (exit) order only if there is a current order is going on (main order is not empty / none)
+            # Also check if order status is complete
+            if main_order is not None and main_order.get_order_status() is BrokerOrderStatusConstants.COMPLETE:
+
+                # Get crossover value
+                crossover = self.get_crossover_value(instrument)
+
+                if (crossover == 1 and self.main_order_map[instrument].order_transaction_type.value == 'SELL') or (crossover == -1 and self.main_order_map[instrument].order_transaction_type.value == 'BUY'):
```

Comment:
instead of
`if (crossover == 1 and self.main_order_map[instrument].order_transaction_type.value == 'SELL') or (crossover == -1 and self.main_order_map[instrument].order_transaction_type.value == 'BUY'):`

we have just done
`if crossover_value in [1, -1]:`

in other strategies.

can we do the same here?

Reply:
done